utopian-io/v2.utopian.io
# Add sentry to auth app

### Verify that
Git checklist

### My PR doesn't concern a Jira task
Implement error tracking on Auth app
---

Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

* Create a new project in your Sentry account
* Add the `SENTRY_DSN` record to your .env
* Lerna bootstrap to add the package
* Trigger an error in the auth app
* The error should appear in the sentry app
